setup.py

```diff
@@ -75,7 +78,7 @@ def write_version_py(version, isreleased, filename):
                         .format(version=version, git_revision=git_revision))
         short_version = version
 
-    with open(filename, 'w') as f:
+    with file_path.open('w') as f:
```

Comment:
Looks like this is because of the difference between a path as a string and a `pathlib.Path`, is this a necessary change or would this work as before? Just curious

Reply:
It would work regardless where the `setup.py` is called from. This was added because, `setup.py `is now being called from `docs/conf.py`, and the relative path specified in the function would lead to an exception.